microsoft-edge/webview2/how-to/self-hosting.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Self-host by deploying preview channels
+description: How to set up self-hosting by deploying using the WebView2 preview channels, which are the Insider preview channels of Microsoft Edge (Canary, Dev, and Beta).
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 02/23/2024
+---
+# Self-host by deploying preview channels
+
+_Self-hosting_ means deploying a preview version of the WebView2 Runtime with your app.  This allows you and your development team to use your app daily with the preview version of the WebView2 Runtime, to find issues that will affect your particular WebView2 app, before the forthcoming changes reach the Stable Evergreen WebView2 Runtime.
```

Comment:
> Unless we aren't recommending having an insider/beta version for use outside of their team
We aren't able to recommend this right now, because we don't have an equivalent of a Evergreen WebView2 Runtime for the Beta / Dev channel. We are working with the deployment team to get that set up, however.

So at this moment in time, we are just recommending selfhosting within their team. Once we have Evergreen Dev/Beta that developers can deploy to their insider rings, we will update this document.